### PR TITLE
Optimize Getting Tickets Service

### PR DESCRIPTION
**Pull request by GigiDev7**

### CODE REVIEW

**Comment by GigiDev7** [comment]
its not finished yet

File: server/services/ticket.js [hunk 1]
```diff
@@ -82,7 +82,159 @@ const findTickets = async (queryObject) => {
...
+  //airplane filter
+  let airplaneFilter = {};
+  if (queryObject?.airplane?.in) {
+    airplaneFilter = {
+      "airplane.company": { $in: queryObject.airplane.in.split(",") },
+    };
+  }
+
+  //ticket class filter
+  let ticketClassFilter = {};
```
Comment (Lokki17) [inline server/services/ticket.js]: Why do you need so many filters? Why dont set all conditions in one object?

**Review by Lokki17 — COMMENTED** [review]
Fix remark

File: server/services/ticket.js (hunk 1 above)
Comment (GigiDev7) [reply server/services/ticket.js]: fixed it

**Pull request merged**